manager/controllers/app/suite_test.go

```diff
@@ -79,9 +82,17 @@ var _ = BeforeSuite(func(done Done) {
 		k8sClient, err = client.New(cfg, client.Options{Scheme: scheme.Scheme})
 		Expect(err).ToNot(HaveOccurred())
 	} else {
+		systemNamespaceSelector := fields.SelectorFromSet(fields.Set{"metadata.namespace": utils.GetSystemNamespace()})
+		workerNamespaceSelector := fields.SelectorFromSet(fields.Set{"metadata.namespace": "m4d-blueprints"})
```

Comment:
Do we have a constant for `m4d-blueprints` somewhere that can be used? This would make future refactoring easier.

Reply:
Replaced with constant.